tfx/components/trainer/executor.py

```diff
@@ -166,6 +192,46 @@ def _GetFnArgs(self, input_dict: Dict[Text, List[types.Artifact]],
         # Additional parameters to pass to trainer function.
         **custom_config)
 
+
+  def _CleanServingDir(self, working_dir: Text) -> None:
```

Comment:
let's just pass modelrun path to fn.serving_model_dir (add some comment), and copy the model out of modelrun to model artifact